src/pocketmine/Player.php

```diff
@@ -3241,7 +3241,14 @@ public function handleItemFrameDropItem(ItemFrameDropItemPacket $packet) : bool{
 
 		$tile = $this->level->getTile($this->temporalVector->setComponents($packet->x, $packet->y, $packet->z));
 		if($tile instanceof ItemFrame){
+			$ev = new PlayerInteractEvent($this, $this->inventory->getItemInHand(), $tile->getBlock(), $tile->getBlock()->getDamage(), PlayerInteractEvent::LEFT_CLICK_BLOCK);
```

Comment:
@SOF3 not really... it _is_ a left-click, exactly as it says. People who misinterpret that, well, it's their own problem.